src/kubernetes.cc

```diff
@@ -1157,12 +1160,15 @@ void KubernetesUpdater::StartUpdater() {
         config().KubernetesClusterLevelMetadata() ? "" : current_node);
 
     // Wrap the bind expression into a function to use as a bind argument.
```

Comment:
Update this comment to mention lambda expression rather than bind.

Reply:
The comment is outdated. I just removed it.